hack/lib/common-var-init.sh

```diff
@@ -44,6 +44,9 @@ export ALLOW_PRIVILEGED=true
 # onebox has option to choose cni plugin: bridge(default), alktron(neutron integration), mizar
 CNIPLUGIN=${CNIPLUGIN:-"bridge"}
 
+VPC_RANGE_START=${VPC_RANGE_START:-11}
+VPC_RANGE_END=${VPC_RANGE_END:-99}
+
```

Comment:
Is this only for mizar or all network cni?

Reply:
My understanding is this is only for mizar use, but the parameters are transferred to kube-controller-manager.  Whether it is really used or not, The parameter 'MizarVPCRangeOverlap' inside "FEATURE_GATES" controls it.  When cniplugin=mizar, "MizarVPCRangeOverlap=true" is added into "FEATURE_GATES" and these two variables are really used; when cniplugin=flannel or default, "MizarVPCRangeOverlap=true" is not added into "FEATURE_GATES".  Because the default value of "MizarVPCRangeOverlap" is "false", these two variables are not really used.

FEATURE_GATES="${FEATURE_GATES},MizarVPCRangeOverlap=true"

Reply:
> My understanding is this is only for mizar use, but the parameters are transferred to kube-controller-manager. Whether it is really used or not, The parameter 'MizarVPCRangeOverlap' inside "FEATURE_GATES" controls it. When cniplugin=mizar, "MizarVPCRangeOverlap=true" is added into "FEATURE_GATES" and these two variables are really used; when cniplugin=flannel or default, "MizarVPCRangeOverlap=true" is not added into "FEATURE_GATES". Because the default value of "MizarVPCRangeOverlap" is "false", these two variables are not really used.
> 
> FEATURE_GATES="${FEATURE_GATES},MizarVPCRangeOverlap=true"

Correct!